apps/dashboard/config/configuration_singleton.rb

```diff
@@ -267,11 +267,16 @@ def config
   end
 
   def read_config
-    f = Configuration.config_file
-    YAML.safe_load(File.open(f.to_s)).symbolize_keys
-  rescue => e
-    Rails.logger.error("Can't read or parse #{Configuration.config_file} because of error #{e}")
-    {}
+    files = Pathname.glob([config_directory.join("*.{yml,yaml}"), config_directory.join("*.{yml,yaml}.erb")])
+    files.each_with_object({}) do |f, config|
+      begin
+        content = ERB.new(f.read, nil, "-").result(binding)
```

Comment:
Can open an issue to track separately, but the context for rendering this ERB should likely include some useful values, perhaps from OodSupport. For example, imagine the context when rendering for me included `username` and `groups` method:

```
username # => "efranz"
groups #=> ["PZS0714", "PZS0002", "PZS0530", "abaqus", "hpcsoft"... "PAS1788"]
```

or groups is the OodSupport group object where you have access to both group and id (and should still be able to do something like `groups.include?("abaqus")`

Reply:
Yea something like #759?

Reply:
I opened https://github.com/OSC/ondemand/issues/906. #759 is a little bit more wide open and likely more work. #906's solution could contribute to #759 which in some since is a refactor step though I think #759 you were wanting to add somethings to the context of rendering that we currently do not have.